Mathlib/Topology/Instances/AddCircle.lean

```diff
@@ -317,6 +348,12 @@ theorem equivAddCircle_symm_apply_mk (hp : p ≠ 0) (hq : q ≠ 0) (x : 𝕜) :
   rfl
 #align add_circle.equiv_add_circle_symm_apply_mk AddCircle.equivAddCircle_symm_apply_mk
 
+/-- The rescaling equivalence between additive circles with different periods. -/
+def homeomorphAddCircle (hp : p ≠ 0) (hq : q ≠ 0) : AddCircle p ≃ₜ AddCircle q :=
+  ⟨equivAddCircle p q hp hq,
+    (continuous_quotient_mk'.comp (continuous_mul_right (p⁻¹ * q))).quotient_lift _,
+    (continuous_quotient_mk'.comp (continuous_mul_right (q⁻¹ * p))).quotient_lift _⟩
+
```

Comment:
Maybe it might be useful to add some rfl-lemmas here. (I see that you have a `change` in the other file where you apply this; I think that whenever you're having to use `change`, it's generally worth asking if that's a sign that some API is missing.)